hub/models.py

```diff
@@ -814,11 +901,39 @@ def get_member_ids_from_webhook(self, payload: dict) -> list[str]:
             "Get member ID not implemented for this data source type."
         )
 
-    async def ingest(self):
+    def import_all(self):
         """
         Copy data to this database for use in dashboarding features.
         """
-        raise NotImplementedError("Ingest not implemented for this data source type.")
+
+        # A LIH record of this data
+        data_set, created = DataSet.objects.update_or_create(
+            external_data_source=self,
+            defaults={
+                "name": self.name,
+                "data_type": "json",
+                "table": "commondata",
+                "default_value": {},
+                "is_filterable": True,
+                "is_shadable": False,
+                "is_public": False,
+            },
+        )
+
+        data_type, created = DataType.objects.update_or_create(
+            data_set=data_set, name=self.id, defaults={"data_type": "json"}
+        )
+
+        data = async_to_sync(self.fetch_all)()
+        for record in data:
+            # To allow us to lean on LIH's geo-analytics features,
+            # TODO: Re-implement this data as `AreaData`, linking each datum to an Area/AreaType as per `self.geography_column` and `self.geography_column_type`.
```

Comment:
Is the reason for doing this sufficiently important to take an issue to capture this TODO, but no important enough for it to be an issue? Feels like it should be an issue. If so, let's take an issue and remove the comment. Otherwise I always feel these TODOs are never done and this feels to me like a pretty important thing to capture.